src/components/KCatalog/KCatalogItem.vue

```diff
@@ -69,8 +69,8 @@ const handleCardClick = (evt: Event, item: CatalogItem): void => {
   overflow: hidden;
 
   .multi-line-truncate {
-    line-height: var(--lh, var(--kui-line-height-30, $kui-line-height-30));
-    max-height: calc(var(--lh, var(--kui-line-height-30, $kui-line-height-30)) * var(--max-lines, 5));
+    line-height: var(--lh, 20px);
+    max-height: calc(var(--lh, 20px) * var(--max-lines, 5));
```

Comment:
question: what are these css custom properties...?

Reply:
They don't follow the standard theming pattern, so I'd say remove them. There are also some in KPopover (or KTooltip, can't remember) -- are they created by us? If yes, let's remove if it doesn't break anything. 

If they are from the popper.js or something, then leave in place

Reply:
Removed these variables. Both KPopover and KTooltip follow the standard theming pattern - but if we find other Kongponents using non-standard variables we can remove them later.